Fig.Tests/IniFileSettingsSourceTests.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.IO;
+using Fig.Core;
+using NUnit.Framework;
+
+namespace Fig.Test
+{
+    /// <summary>
+    /// Tests to prove that INI Files are correctly parsed
+    /// </summary>
+    public class IniFileSettingsSourceTests
+    {
+        private IniFileSettingsSource _iniFileSettingsSource;
+        private string _testFilePath;
+
+        [SetUp]
+        public void Setup()
+        {
+            // Temporary test file path
+            _testFilePath = $"{Path.GetTempPath()}IniFileSettingsSourceTest_{System.DateTime.Now.Ticks}.ini";
+
+            // Generate test Ini File
+            File.WriteAllText(_testFilePath, String.Join(Environment.NewLine, new string[] {
```

Comment:
I would have added an internal constructor to the IniFileSettingsSource that takes a `string[] lines` argument. This would eliminate the need to deal with file handling in the test

Reply:
I like that.  I debated whether to add something like a FileStream to the public constructor, but your idea works.